src/components/layout/MobileNavbar.jsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import { graphql, StaticQuery, Link } from 'gatsby';
+import styled from 'styled-components';
+import { CSSTransition } from 'react-transition-group';
+
+class MobileNavBar extends React.Component {
+  state = {
+    active: null,
+    showSubMenu: false,
+  };
+
+  handleMenuClick = clicked => {
+    clicked === this.state.active
+      ? this.setState({ showSubMenu: false })
+      : this.setState({ active: clicked, showSubMenu: true });
+  };
+
+  renderMenuItems = pages =>
+    pages.map(menuItem => {
+      if (menuItem.node.parentPage === null) {
+        return (
+          <ParentMenuItem
+            key={menuItem.node.title}
+            onClick={() => this.handleMenuClick(menuItem.node.title)}
+            active={this.state.active === menuItem.node.title}
+          >
+            {menuItem.node.title}
+          </ParentMenuItem>
+        );
+      }
+      return null;
+    });
+
+  renderSubMenuItems = pages =>
+    pages.map(menuItem => {
+      if (
+        menuItem.node.parentPage !== null &&
+        menuItem.node.parentPage.title === this.state.active
+      ) {
+        return (
+          <Link
+            to={'/' + menuItem.node.parentPage.slug + '/' + menuItem.node.slug}
+            key={menuItem.node.title}
+          >
+            <div className="sub-menu-item">{menuItem.node.title}</div>
+          </Link>
+        );
+      }
+      return null;
```

Comment:
Same comment as for renderMenuItems

Reply:
Fixed in [94379c0](https://github.com/svsticky/static-sticky/pull/67/commits/94379c02adb1c577a6b81ed9ff92da33201c58e9)